Storage/SQL/SQLiteHistory.swift

```diff
@@ -299,9 +299,7 @@ extension SQLiteHistory: BrowserHistory {
         let (query, args) = self.filteredSitesByFrecencyQueryWithHistoryLimit(limit, bookmarksLimit: 0, groupClause: groupBy, whereData: whereData)
         let cacheArgs: Args = [limit]
 
-        return accumulate([
-            { self.db.runQuery(query, args: args, factory: SQLiteHistory.iconHistoryColumnFactory) },
```

Comment:
Was this throwing a linter warning? Having it all on one line makes it really long and harder to read.